ext/rbs_extension/parser.c

```diff
@@ -1836,14 +1839,18 @@ VALUE parse_mixin_member(parserstate *state, bool from_interface, position comme
   rbs_loc_add_required_child(loc, rb_intern("keyword"), keyword_range);
   rbs_loc_add_optional_child(loc, rb_intern("args"), args_range);
 
-  return rbs_ast_members_mixin(
```

Comment:
Likewise, we split this one constructor function (`rbs_ast_members_mixin()`) into 3 new functions:

1. `rbs_ast_members_extend()`
2. `rbs_ast_members_include()`
3. `rbs_ast_members_prepend()`

These will be modelled by 3 different C structs, to match the 3 different Ruby classes that exist for them.